templates/_layouts/docs.html

```diff
@@ -74,7 +74,6 @@
               {{ side_nav_item("/docs/patterns/heading-icon", "Heading icon") }}
               {{ side_nav_item("/docs/patterns/icons", "Icons") }}
               {{ side_nav_item("/docs/patterns/images", "Images") }}
-              {{ side_nav_item("/docs/patterns/labels", "Labels") }}
```

Comment:
We remove this page, so the contents (`labels.md`) file should be removed as well.

Reply:
I simply renamed the file (https://github.com/canonical-web-and-design/vanilla-framework/pull/4400/files#diff-b5623168757e725faf8450740b0d9cdf3fe879d342b8f72c8f5dd0ac4619f674), so I think `labels.md` it's already deleted